wrappers/javascript/aries-askar-react-native/native/aries_askar.node

```diff
@@ -1,4 +0,0 @@
-# This is a placeholder file to prevent node-pre-gyp from installing the
-# aries-askar binary when cloning this repository. It won't be published to
```

Comment:
This file should not be removed I think?

Otherwise when you run yarn install it will automatically download the binaries, which you may not want

Reply:
For the React Native Example to be used without building it locally it is required now. It is not optimal, agreed.

Reply:
I think it'd fail if we increase the native binary version. As the CI will then try to install the native binary, but it doesn't exist yet

Reply:
Being required to build it locally doesn't seem like a bad thing to me, is there a reason we don't want to do that?

Reply:
The main reason right now is that `cross` only allows to be ran on a Linux host which would make building the application for Android on macOS or Windows impossible. With a better example app we could add something where you specify the platform and you can choose to only build the application for iOS which would make it work on macOS.